lib/membrane/rtp/serializer.ex

```diff
@@ -10,16 +10,14 @@ defmodule Membrane.RTP.Serializer do
   use Membrane.Filter
 
   require Bitwise
-  alias Membrane.{RemoteStream, RTP}
+  alias Membrane.RTP
 
   @max_seq_num Bitwise.bsl(1, 16) - 1
   @max_timestamp Bitwise.bsl(1, 32) - 1
 
   def_input_pad :input, caps: RTP, demand_mode: :auto
 
-  def_output_pad :output,
-    caps: {RemoteStream, type: :packetized, content_format: RTP},
-    demand_mode: :auto
+  def_output_pad :output, caps: RTP, demand_mode: :auto
```

Comment:
Such change kind of breaks the concept of caps - in theory, the `RemoteStream` has all the headers in the payload and isn't parsed while RTP contains just RTP payload as `Membrane.Buffer`'s payload. This is also symmetrical to the `RTP.Parser` that does the opposite conversion and changes caps from `RemoteStream` to `RTP`.
So the question is - what is the purpose of this change? I may have missed some conversation about it, but if the `RTP` caps are needed here for some reason, we may need more complex changes to the caps inside the RTP plugin. Maybe we should eliminate RemoteStream caps (at least for output) and have some distinction of payloads as a field inside `RTP` caps? Although I guess this solution was rejected at some point when adding RemoteCaps.